layers/+tools/lsp/funcs.el

```diff
@@ -25,6 +25,23 @@
     (setq key (pop bindings)
           def (pop bindings))))
 
+(defun spacemacs/lsp-bind-upstream-keys ()
```

Comment:
Here you can use `lsp-enable-which-key-integration` in `(let ((lsp-prefix-key "..."))` block.

Reply:
Hmm good point, but we should add it in a later PR